DataMigration/src/test/java/org/sagebionetworks/dian/datamigration/PathsHelperTests.java

```diff
@@ -163,9 +163,7 @@ public void test_findAllJsonFilesInFolder() throws IOException {
         files.sort((o1, o2) -> o1.getFileName().toString()
                 .compareTo(o2.getFileName().toString()));
         assertEquals("a.json", files.get(0).getFileName().toString());
-        assertEquals("FolderC", files.get(0).getParent().getFileName().toString());
```

Comment:
@brucehoff Github updated the version of Ubuntu the unit tests run on, this changed the sort order of both "a.json" files.  The important part is they are both there, not which one comes first.